api/valor_api/main.py

```diff
@@ -387,8 +392,8 @@ def get_datasets(
 
     Parameters
     ----------
-    filters : schemas.Filter, optional
-        An optional filter to constrain results by.
+    filters : schemas.FilterQueryParams, optional
+        An optional filter to constrain results by. All of these fields should be json strings
```

Comment:
```suggestion
        An optional filter to constrain results by. All fields should be specified as strings in a JSON.
```

Reply:
the two big suggestions here are a) capitalize JSON for consistency and b) add a period at the end of the sentence

Reply:
yep. done